src/resources/MachineLearning/SESConfiguration/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './SESConfiguration.js';
```

Comment:
Exporting the new resource is not enough, you need to bind it to the platform client instance, otherwise you won't be able to call it using `Platform.ml.SESConfiguration`

Reply:
See [James PR](https://github.com/coveo/platform-client/pull/775/files#diff-e14f31736689141658844128c7b748e18642ee7ca51e0de71ba25fe5e41a649c) for an example of how to do that.

Reply:
fixed